appveyor.yml

```diff
@@ -69,19 +69,29 @@
 
             choco install haskellplatform -version 2014.2.0.0 -y
 
-            $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine")
+            # choco install updated the path, so re-read them from the registry and reset $env:path
+
+            $userPath = [System.Environment]::GetEnvironmentVariable("Path","User")
+
+            $machinePath = [System.Environment]::GetEnvironmentVariable("Path","Machine")
+
+            $env:Path = "$userPath;$machinePath"
 
             cabal update
 
             if ($env:BOND_BUILD -eq "Doc") {
 
-                cabal install -j pandoc --constraint='mtl<=2.1.3.1'
+                choco install pandoc --version 1.19.2  -y
 
                 choco install doxygen.install -y
 
-            }
+                $userPath = [System.Environment]::GetEnvironmentVariable("Path","User")
```

Comment:
Should these lines move inside the if-block or does cabal update also change the path?